plugins/woocommerce/src/Enums/CatalogVisibility.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Enums;
+
+/**
+ * Enum class for all the catalog visibility values.
+ */
+class CatalogVisibility {
```

Comment:
Shouldn't it be final?

Reply:
Right! Updated it on https://github.com/woocommerce/woocommerce/pull/54001/commits/168958b2e244b0141bbe56163f999e2af633d842